src/Containers/CadView.jsx

```diff
@@ -214,6 +214,7 @@ export default function CadView({
     createPlaceMark({
       context: viewer.context,
       oppositeObjects: [tmpModelRef],
+      postProcessor: viewer.postProcessor,
```

Comment:
Hi @Ibrahim5aad, nice to see `CustomPostProcessing` functionality change.
could you share a link for post processor of ifc viewer?
I need to look around it a bit.

Reply:
Hi @ronplaymeta, sorry I'm not able to get what do you mean by a link for it?
There are three places where post processing is applied:

First in selection, you could spot a yellow outline on selected elements like so: 
![image](https://user-images.githubusercontent.com/50090593/226044929-b8035d8d-cd21-40cb-9b9a-525becca2744.png)

Second by temporary isolating an element by clicking "i" while selecting an elements:

![image](https://user-images.githubusercontent.com/50090593/226045519-dddc5d63-d117-4893-98bf-c2b6cc3077bf.png)


the third one is not added by me and I don't think it is not available through UI atm.